definitions/common/json/CaseField/CaseField-common.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "01/06/2020",
+    "CaseTypeID": "${CCD_DEF_CASE_TYPE_ID}",
+    "ID": "generalLetterPreview",
```

Comment:
How come we are only migrating the generalLetterPreview Document field across to common components? Is this just our first tester field to check it works?

Reply:
exactly